### ISSUE TITLE
User scripts aren't in menu bar

### ISSUE THREAD

**Issue by patrik-krajcovic2** [xiaoxiaoflood/firefox-scripts#203]
Actual results:
User scripts are missing from menu bar.

Expected results:
User scripts are in menu bar.

**Comment by patrik-krajcovic2** [comment]
No, I think menu bar, you don't know what is menu bar? I don't think menu.

**Comment by patrik-krajcovic2** [comment]
Menu bar is this.

![Menu bar](https://user-images.githubusercontent.com/69351811/201048017-cad8c9e6-a8d0-48ec-b260-354b152f37a7.png)

**Comment by Lalarian** [comment]
You mean the 'userChromeJS Manager'? Die you install this, as it's a separate script!?
It usually is shown either in the icon bar OR the menu, not both at once.
If you can't find it anywhere, it might be in the icon bar's 'storage': Rigth click the icon bar, choose 'customize icons' or whatever it is called in englisch, and check if the icon is there. From there you can drag and drop it to the icon bar.
It'slooking like this:
![Bildschirmfoto_2023-01-23_13-23-22](https://user-images.githubusercontent.com/5146616/214039298-dda0b10a-e95b-4d3e-b417-aa050d552ac0.png)

**Comment by Patricek0056** [comment]
> You mean the 'userChromeJS Manager'? Did you install this?
Yes, I meant that, Yes I have it installed. I opened it, because they are missing from menu bar, but they are in toolbar or application menu.

**Comment by eassun** [comment]
My UC Scripts also disappeared from the toolbar. I can't find the icon by customizing the toolbar. I really want to know how to move the icon for UC Scripts from the application menu to the toolbar. Thanks! ! !

**Comment by Patricek0056** [comment]
Hey! This is not about toolbar but **MENU BAR**. Menu bar with File, Edit, View, History, Bookmarks, Tools and Help. They are not in **MENU BAR**.

**Comment by xiaoxiaoflood** [comment]
userChromeJS Manager is displayed as toolbar button **OR** as submenu in menubar → Tools depending on the value of `userChromeJS.showtoolbutton`, never both. If the value of that pref is `false`, you'll see the button but not the menu. If it's `true`, the opposite.

Other than that, the script doesn't control the placement of the toolbar button, it's created and handled by Firefox just like any other toolbar button.

The screenshot below was taken with `userChromeJS.showtoolbutton = true`, so with submenu in menubar but without toolbar button.

![image](https://user-images.githubusercontent.com/25161346/220503858-e187a984-71b1-4baa-86f3-2518640119a5.png)

**Comment by aaronedev** [comment]
i clicked the button "hid from toolbar" i believe and now the button is in application menu - however i want to undo or move it back to the toolbar. Is there no such option?

**Comment by aaronedev** [comment]
ok nevermind - you gotta show the menu bar and go under tools to undo
